cmd/plc-server/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/stellentus/go-plc"
+)
+
+var (
+	plcAddr  = flag.String("plc-address", "192.168.1.176", "Hostname or IP address of the PLC")
+	path     = flag.String("path", "1,0", "Path to the PLC at the provided host or IP")
+	httpAddr = flag.String("http", ":8784", "Port for http server to listen to")
+)
+
+var knownTags = map[string]interface{}{
+	"Program:ZW.B801.INISCP":      uint32(0),
+	"Program:ZW.B801.MAXRANGE_SP": uint32(0),
+	"Program:ZW.B801.SPEED_SP_1":  uint32(0),
+	"Program:ZW.B801.SPEED_SP_2":  uint32(0),
+}
+
+func main() {
+	flag.Parse()
+
+	connectionInfo := fmt.Sprintf("protocol=ab_eip&gateway=%s&path=%s&cpu=controllogix", *plcAddr, *path)
+	timeout := 5 * time.Second
+	device, err := plc.NewDevice(connectionInfo, timeout)
+	panicIfError(err, "Could not create test PLC!")
+	defer func() {
+		err := device.Close()
+		if err != nil {
+			fmt.Println("Close was unsuccessful:", err.Error())
+		}
+	}()
+
+	http.Handle("/tags/raw", RawTagsHandler{device, knownTags})
```

Comment:
Should we namespace the API (ie. this is `/v1/tags/raw`)?

Reply:
I'm inclined to move all of the code in `cmd` into `example` (or vice versa) and treating all of them as example code.
When we get to writing our real server, we'll probably want to put it into a private repo. I'm not totally committed to this approach, but @matthias-stone has been pushing for it.
So if we see this as merely an example, the namespace isn't important.

Reply:
For my part, I see the value of `go-plc` being an external dependency on the HTTP service codebase in a different repo, but I don't feel strongly about one approach vs the other.

Reply:
I'm good with having this with other examples.